modules/dnn/test/test_layers.cpp

```diff
@@ -951,12 +956,15 @@ TEST(Layer_Test_Convolution_DLDT, setInput_uint8)
     for (int i = 0; i < 2; ++i)
     {
         Net net = readNet(_tf("layer_convolution.xml"), _tf("layer_convolution.bin"));
+        net.setPreferableTarget(targetId);
         net.setInput(inputs[i]);
```

Comment:
setPreferableBackend?

Reply:
It turns automatically to `DNN_BACKEND_INFERENCE_ENGINE` for models in Intermediate Representation (.xml and .bin).